src/main/java/fr/openent/homeworkAssistance/models/FeaturedResources.java

```diff
@@ -0,0 +1,70 @@
+package fr.openent.homeworkAssistance.models;
+
+public class FeaturedResources {
+    private String idRessource;
+    private String nomRessource;
+    private String urlVignette;
+    private String urlAccesRessource;
+    private String description;
+
+    public FeaturedResources(String idRessource, String nomRessource, String urlVignette, String urlAccesRessource, String description) {
+        this.idRessource = idRessource;
+        this.nomRessource = nomRessource;
+        this.urlVignette = urlVignette;
+        this.urlAccesRessource = urlAccesRessource;
+        this.description = description;
+    }
+
+    // Getters and Setters
+    public String getIdRessource() {
+        return idRessource;
+    }
+
+    public void setIdRessource(String idRessource) {
+        this.idRessource = idRessource;
+    }
+
+    public String getNomRessource() {
+        return nomRessource;
+    }
+
+    public void setNomRessource(String nomRessource) {
+        this.nomRessource = nomRessource;
+    }
+
+    public String getUrlVignette() {
+        return urlVignette;
+    }
+
+    public void setUrlVignette(String urlVignette) {
+        this.urlVignette = urlVignette;
+    }
+
+    public String getUrlAccesRessource() {
+        return urlAccesRessource;
+    }
+
+    public void setUrlAccesRessource(String urlAccesRessource) {
+        this.urlAccesRessource = urlAccesRessource;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    @Override
+    public String toString() {
+        return "Resource{" +
+                "idRessource='" + idRessource + '\'' +
+                ", nomRessource='" + nomRessource + '\'' +
+                ", urlVignette='" + urlVignette + '\'' +
+                ", urlAccesRessource='" + urlAccesRessource + '\'' +
+                ", description='" + description + '\'' +
+                '}';
+    }
```

Comment:
si tu te sers pas de la fonction tu peux la suppr
et sinon utilise des constants ici pour tes noms de champs
et pour le nom de ta class au début, pour être safe l'idéal serait d'utiliser un this.getClass().getSimpleName() dans ce cas au lieu de "Resource" (qui n'est d'ailleurs pas le nom de ta classe justement)